.circleci/config.yml

```diff
@@ -770,6 +770,7 @@ jobs:
     environment:
       LANG: "C.UTF-8"
       EMTEST_SKIP_V8: "1"
+      EMCC_CORES: "4"
```

Comment:
Would it make more sense to modify the linux-python executor instead? Other cores/runner settings seem to be up there.

Reply:
Done.